EventFilter/LumiRawToDigi/plugins/LumiRawToDigi.cc

```diff
@@ -0,0 +1,131 @@
+// -*- C++ -*-
+//
+// Package:    ProdTutorial/LumiRawToDigi
+// Class:      LumiRawToDigi
+// 
+/**\class LumiRawToDigi LumiRawToDigi.cc ProdTutorial/LumiRawToDigi/plugins/LumiRawToDigi.cc
+
+ Description:  Producer to unpack lumi nibble from TCDS 
+
+*/
+//
+// Original Author:  Chris Palmer
+//         Created:  Thu, 28 May 2015 19:54:56 GMT
+//
+//
+
+
+// system include files
+#include <memory>
+#include <iostream>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDProducer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+
+#include "DataFormats/FEDRawData/interface/FEDRawDataCollection.h"
+#include "DataFormats/FEDRawData/interface/FEDHeader.h"
+#include "DataFormats/FEDRawData/interface/FEDTrailer.h"
+#include "DataFormats/FEDRawData/interface/FEDNumbering.h"
+
+#include "EventFilter/FEDInterface/interface/GlobalEventNumber.h"
+#include "EventFilter/FEDInterface/interface/FED1024.h"
+
+#include "DataFormats/FEDRawData/interface/FEDRawData.h"
+
+
+using namespace std;
+
+//
+// class declaration
+//
+
+class LumiRawToDigi : public edm::EDProducer {
```

Comment:
class LumiRawToDigi : public edm::EDProducer { 
to
class LumiRawToDigi : public stream::EDProducer<> {
causes a compiler error.

 error: 'stream' has not been declared
 class LumiRawToDigi : public stream::EDProducer<> {

Do I need to add an include?